admin_guide/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -172,3 +172,29 @@ Persistent volume claims only exist in the user's namespace and can only be
 referenced by a pod within that same namespace. Any attempt to access a
 persistent volume from a different namespace causes the pod to fail.
 ====
+
```

Comment:
You can kill the above [IMPORTANT] box given the addition of the "Volume Security" section.